### DISCUSSION TITLE
Error on npx create

### DISCUSSION THREAD

**Discussion by kypanz** [AlbericoD/overwolf-modern-react-boilerplate#174] [General]
**Describe the bug**
Muliple errors when i try to install using the overwolf template

**To Reproduce**

Install with : npx create-react-app my-app --template overwolf-typescript-redux

**Screenshots**

https://cdn.discordapp.com/attachments/501360270363590656/1005168584173035590/unknown.png

**Desktop (please complete the following information):**
nodejs : v16.14.2
npm : 8.5.0

**Comment by AlbericoD** [comment]
Hi, I couldn't open the image, could you write down the error?

**Comment by bmitrano825** [comment]
> Hi, I couldn't open the image, could you write down the error?

I was able to open it by highlighting the entire link and then copying and pasting in my browser. 

I am also having the same issue Kypanz is having.

**Comment by kypanz** [comment]
Hello guys
- yes, copy and pasting the link in to the browser and you can see the error
- the solution for this error is => in the installation node for "windows" you need to "check" the nodejs values : https://prnt.sc/GBo-8MjfRySx

- result after "check" the nodejs values => https://prnt.sc/CS-dsGPvd3fu

note : i update the links in this issue, so now the links works fine, if this work please like my comment ♥, with that i know this works for the community ♥

**Comment by Munaf-Divan** [comment]
Hello @kypanz, even after checking that button while node installation in Windows, I am getting the same error; I am attaching an image for your reference.

Environment: Windows 10 pro, Nodejs: 20.9.0, npx: 10.1.0, create-react-app: 5.0.1

![image](https://github.com/AlbericoD/overwolf-modern-react-boilerplate/assets/140388383/ea7d5669-a813-4644-bd53-bce3b406dc5e)
